tools/IncrementVersion.ps1

```diff
@@ -1,21 +1,55 @@
-$versionPath = "$PSScriptRoot\CustomMSBuild\Versioning.props"
+<#
+.PARAMETER versionPath
+  The path to the msbuild props file where the version number is specified
+.PARAMETER lastReleaseCommit
+  The ID of the last commit to be released
+.PARAMETER forceMinorIncrement
+  Whether to force an increment of the minor version number, regardless of API changes since the last release
+.PARAMETER forceRevisionIncrement
+  Whether to force an increment of the revision version number, regardless of API changes since the last release
+#>
```

Comment:
Does this script has a help option with examples on how to use the parameters?

Reply:
I have not provided examples, but `help .\tools\IncrementVersion.ps1 -detailed` does use these descriptions:

NAME
    C:\github\odata.net\tools\IncrementVersion.ps1

SYNOPSIS


SYNTAX
    C:\github\odata.net\tools\IncrementVersion.ps1 [[-versionPath] <String>] [[-lastReleaseCommit] <String>]
    [-forceMinorIncrement] [-forceRevisionIncrement] [<CommonParameters>]


DESCRIPTION


PARAMETERS
    -versionPath <String>
        The path to the msbuild props file where the version number is specified

    -lastReleaseCommit <String>
        The ID of the last commit to be released

    -forceMinorIncrement [<SwitchParameter>]
        Whether to force an increment of the minor version number, regardless of API changes since the last release

    -forceRevisionIncrement [<SwitchParameter>]
        Whether to force an increment of the revision version number, regardless of API changes since the last release

    <CommonParameters>
        This cmdlet supports the common parameters: Verbose, Debug,
        ErrorAction, ErrorVariable, WarningAction, WarningVariable,
        OutBuffer, PipelineVariable, and OutVariable. For more information, see
        about_CommonParameters (https:/go.microsoft.com/fwlink/?LinkID=113216).

REMARKS
    To see the examples, type: "get-help C:\github\odata.net\tools\IncrementVersion.ps1 -examples".
    For more information, type: "get-help C:\github\odata.net\tools\IncrementVersion.ps1 -detailed".
    For technical information, type: "get-help C:\github\odata.net\tools\IncrementVersion.ps1 -full".